api/persistence.go

```diff
@@ -0,0 +1,22 @@
+package api
+
+import (
+	"encoding/json"
+	"syscall"
+)
+
+
+type Persistence interface {
+	// [new and lock the filelock and] store the just created sandbox
+	NewSandbox(*Sandbox) error
+	// NewContainer(*Container) error
+	// [lock the filelock and] build sandbox from existing running sandbox
+	GetSandbox(id string, SandboxBuilder) (*Sandbox, error)
+	// realese sandbox [and unlock the filelock]
+	PutSandbox(*Sandbox) error
+
+	// queries
+	ListSandboxs()[]SandboxStatus
+	ListContainers()[]ContainerStatus
+	SandboxOfContainer(string)string
```

Comment:
What does this do?

Reply:
`kata-runtime kill container-name`
we need to find the sandbox from the container-name at first.
In runv, there is a symlink in the container's state root.